dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/osgi/mbeans/PluginGenerator.java

```diff
@@ -775,6 +782,7 @@ protected synchronized void generateXML(String rootLoc, String serverName,
 
             // Only write out to file if we have new or changed configuration information, or if this is an explicit request
             if (writeFile || !utilityRequest || !fileExists) {
+                // Bundle must be valid in order to write the file
```

Comment:
We're just checking that the bundle is not uninstalled, can be correct this comment to say that? `The bundle must not be uninstalled in order to write the file` or something like that. If this isn't actually the case let me know and we can discuss.

Reply:
I can see that this can cause some confusion. Let me update it